Python/errors.c

```diff
@@ -52,6 +52,15 @@ PyErr_Restore(PyObject *type, PyObject *value, PyObject *traceback)
     Py_XDECREF(oldtraceback);
 }
 
+PyExcState *_PyErr_GetExcInfo(PyThreadState *tstate) {
```

Comment:
The result of this function is read-only, and exc_previous is not used. Wouldn't be better to return just three objects like in `PyErr_GetExcInfo()`? This would make the `PyExcState` structure more private.

Reply:
If I made it return three objects, it would be the same as `PyErr_GetExcInfo()`.
`_PyErr_GetExcInfo()` is provided as a way to handle the `type, value, traceback` trio in a reasonably modular fashion, without to much overhead.
As it is only used in `errors.c` and `sysmodule.c`, it is effectively private.

Reply:
If it would be the same as `PyErr_GetExcInfo()`, we should just use `PyErr_GetExcInfo()`.